bootstrapper/internal/initserver/initserver_test.go

```diff
@@ -402,6 +406,10 @@ func (d *stubDisk) UpdatePassphrase(string) error {
 	return d.updatePassphraseErr
 }
 
+func (d *stubDisk) MarkDiskForReset() error {
+	return nil
+}
```

Comment:
Could we add a test case that asserts MarkDisk is called (or that the right error type is returned, see other thread) on non-recoverable error states?